docs/reference/modules/cluster/allocation_awareness.asciidoc

```diff
@@ -44,7 +44,9 @@ nodes.
 
 Now, if we start two more nodes with `node.rack_id` set to `rack_two`,
 Elasticsearch will move shards across to the new nodes, ensuring (if possible)
-that the primary and replica shards are never in the same rack.
+that no two copies of the same shard will be in the same rack. However if `rack_two`
+was to fail, bringing down both nodes on it, Elasticsearch will still allocate the lost
```

Comment:
"was to fail, bringing down both nodes on it" -> "were to fail, taking down both of its nodes"